pom.xml

```diff
@@ -181,6 +181,10 @@
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>cloudwatchlogs</artifactId>
         </dependency>
+        <dependency>
```

Comment:
why we need to add this dependency?

Reply:
Because to test with actual SDK class like exinsting tests (e.g. `cloudwatchlogs.PutTargetsRequest`).
https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/429/files#diff-0df6d780ae71a8519e522b0d64d0792c1c10362f9b8bd9c0c3e983cf741ac6a9R62-R72

I should test using created class for test only instead new dependency?

Reply:
I change scope of dependency to test.